config/bq-piccolo.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+# DTB: From https://dl.twrp.me/piccolo/twrp-3.7.0_9-0-piccolo.img
+
+OPTIONS=(-r power --backlight-gpio)
```

Comment:
Please add `--dumb-dcs` so the panel driver doesn't look so incredibly weird (also I think the generator warns about this). It doesn't look like the panel driver uses DCS according to the specification.